src/Api/SecretsManager/Models/Request/ServiceAccountsAccessPoliciesRequestModel.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Bit.Core.Exceptions;
+using Bit.Core.SecretsManager.Entities;
+using Bit.Core.SecretsManager.Models.Data;
+
+namespace Bit.Api.SecretsManager.Models.Request;
+
+public class ServiceAccountsAccessPoliciesRequestModel
+{
+    public IEnumerable<AccessPolicyRequest> ProjectServiceAccountsAccessPolicyRequests { get; set; }
+
+    private static void CheckForDistinctAccessPolicies(IReadOnlyCollection<BaseAccessPolicy> accessPolicies)
```

Comment:
We have a `CheckForDistinctAccessPolicies` in `PeopleAccessPoliciesRequestModel` too. 

What do you think about creating a new static class at `src/Api/SecretsManager/Utilities/AccessPolicyHelpers.cs`?

Then having a generic function, we can call in both places `AccessPolicyHelpers.CheckForDistinctAccessPolicies(policies);`

Open to using different names.

Reply:
Love this idea 😄